src/services/services.ts

```diff
@@ -4648,7 +4648,7 @@ module ts {
                     return true;
                 }
                 else if (parent.kind === SyntaxKind.BinaryExpression && (<BinaryExpression>parent).left === node) {
-                    var operator = (<BinaryExpression>parent).operator;
+                    var operator = (<BinaryExpression>parent).operatorToken.kind;

```

Comment:
Sure!  That exists in C# and VB today.  Although they get it just by leveraging 'find references'.